content/auth/token.textile

```diff
@@ -231,25 +251,16 @@ The process used by Ably SDKs to authenticate with Ably using a @TokenRequest@ i
 
 The following is an example of creating an Ably @TokenRequest@:
 
-```[javascript] 
-const ably = new Ably.Rest({ key: '{{API_KEY}}' });
-ably.auth.createTokenRequest({ clientId: 'client@example.com' }, null, (err, tokenRequest) => {
-  /* tokenRequest => {
-       "capability": "{\"*\":[\"*\"]}",
-       "clientId": "client@example.com",
-       "keyName": "{{API_KEY_NAME}}",
-       "nonce": "5576521221082658",
-       "timestamp": {{MS_SINCE_EPOCH}},
-       "mac": "GZRgXssZDCegRV....EXAMPLE"
-     } */
-});
+```[javascript]
+const ably = new Ably.Rest.Promise({ key: '{{API_KEY}}' });
+const tokenRequest = await ably.auth.createTokenRequest({ clientId: 'client@example.com' }, null);
```

Comment:
Just so you're clear here: the optional `null` argument was used before because, when using callbacks, the callback has to be the final argument. Since we don't need to pass the callback anymore we don't need to pass this argument either, and we should remove it to make the code look nicer. From a quick search I can see a few more examples where we can do the same
```suggestion
const tokenRequest = await ably.auth.createTokenRequest({ clientId: 'client@example.com' });
```

Reply:
I'm not quite sure why I added the unnecessary `null` argument here tbh. Removed. https://github.com/ably/docs/pull/2119/commits/e90046ac431bd699719987715d1edab1ac28e59d